docs/index.md

```diff
@@ -8,13 +8,13 @@ id: home
   <div class="marketing-row">
     <div class="marketing-col">
       <h3>Declarative</h3>
-      <p>React makes it painless to update your views when your data changes. Just return the views to display for each possible state, and React handles the updates.</p>
```

Comment:
I tried to remove “painless”, “just”, “easily” because these are somewhat subjective.